drivers/dma/dma_iproc_pax_common.h

```diff
@@ -0,0 +1,536 @@
+/*
+ * Copyright 2020 Broadcom
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef DMA_IPROC_PAX_COMMON
+#define DMA_IPROC_PAX_COMMON
+
+/* Broadcom  PAX-DMA RM register defines */
+#define REG_ADDR(_base, _offs) ((_base) + (_offs))
+#define RING_ADDR_OFFSET(_ring) (0x10000 * (_ring))
+
+/* Per-Ring register offsets */
+#define RING_VER 0x000
```

Comment:
Nit: Lining up the values in one column tends to be slightly easier to read, but not a huge deal either way.

Reply:
Also, overall, some of the names here are pretty generic and not namespaced with a prefix (PAX_* or somesuch). It might be useful to add that to avoid conflicts at some point.

Reply:
Sure, let me fix. Thanks

Reply:
Used prefixes based on register category and where necessary PAX_DMA*